src/app/styles/theme/theme.js

```diff
@@ -0,0 +1,9 @@
+import { breakpoints, colors, fontStyle } from "./ui";
+
+const theme = {
+  fontStyle,
+  colors,
+  breakpoints,
+};
+
+export default theme;
```

Comment:
I think we can move it directly to index.js?

Reply:
Yeah, you are right!